File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/glide/GlideActivity.java
package com.jdroid.android.sample.ui.glide;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class GlideActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return GlideFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/glide/GlideActivity.kt
package com.jdroid.android.sample.ui.glide

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class GlideActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return GlideFragment::class.java
    }
}
